lumicks/pylake/channel.py

```diff
@@ -62,6 +62,11 @@ def timestamps(self):
         """Absolute timestamps (since epoch) which correspond to the channel data"""
         return self._src.timestamps
 
+    @property
+    def time(self):
+        """Relative time (in seconds) that corresponds to the channel data"""
```

Comment:
Time for some name bikeshedding :) I think it isn't very obvious that `timestamps` == absolute timestamps in nanoseconds and `time` == relative timestamps in seconds. I would like to make the naming a bit more obvious and differentiate the two properties. Maybe `.relative_time` or `.relative_seconds` would be nice. What do you think?

Reply:
Good point. For an attribute like this that is likely to be used frequently by users (ie, for plotting) I like to keep the name as short as possible which is why I picked `time` originally.  How about just `seconds`? Still different, descriptive and short

Reply:
`.seconds` sounds great!